xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/internal/builder/PresentationBuilderConfiguration.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.officeimporter.internal.builder;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.phase.Initializable;
+import org.xwiki.component.phase.InitializationException;
+import org.xwiki.configuration.ConfigurationSource;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Configuration for the presentation builder.
+ *
+ * @version $Id$
+ * @since 16.8.0
+ * @since 16.4.4
+ * @since 15.10.13
+ */
+@Component(roles = PresentationBuilderConfiguration.class)
+@Singleton
+public class PresentationBuilderConfiguration implements Initializable
+{
+    protected static final String DEFAULT_IMAGE_FORMAT = "jpg";
+
+    private int slideWidth = 1920;
+
+    private float quality = 95;
+
+    @Inject
+    private ConfigurationSource configurationSource;
```

Comment:
Maybe we should use a more specific configuration source here?

Reply:
Should go in xwiki.properties.vm by default, we already have the configuration there for LO. So I'd use `@Named("xwikiproperties")`